hcl/fmt_test.go

```diff
@@ -0,0 +1,224 @@
+// Copyright 2022 Mineiros GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hcl_test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/madlambda/spells/assert"
+	"github.com/mineiros-io/terramate/errors"
+	"github.com/mineiros-io/terramate/hcl"
+	"github.com/mineiros-io/terramate/test"
+
+	errtest "github.com/mineiros-io/terramate/test/errors"
+)
+
+func TestFormatHCL(t *testing.T) {
+	type testcase struct {
+		name     string
+		input    string
+		want     string
+		wantErrs []error
+	}
+
+	tcases := []testcase{
+		{
+			name: "attributes alignment",
+			input: `
```

Comment:
when we start adding extra formatting features the idea is to add more test cases here